components/search_hint/search_hint.tsx

```diff
@@ -46,7 +46,7 @@ const SearchHint = (props: Props) => {
                     <li
                         className={classNames('search-hint__suggestions-list__option', {highlighted: optionIndex === props.highlightedIndex})}
                         key={option.searchTerm}
-                        onClick={() => props.onOptionSelected(option.searchTerm)}
+                        onMouseDown={() => props.onOptionSelected(option.searchTerm)}
```

Comment:
Same here

Reply:
Safari comprehends this handler better as well it seems 🤷‍♀️